_posts/en/2016-xx-xx-measure-operation-time-using-console-object.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+
+title: Measure Operation time using console object
+tip-number: xx
+tip-username: SarjuHansaliya
+tip-username-profile: https://github.com/SarjuHansaliya
+tip-tldr: Measure Operation time using `console` object
+
+categories:
+    - en
+---
+
+Sometime we need to measure time of some tasks to analyze how much time it is taking to execute.To achieve that normal practice is to notedown time before starting of the task and notedown time after completion of the task , and then take a difference between two times.But that is lots of work for only time measuring.
```

Comment:
"how much time it takes to execute"
Space after dot.
"To achieve that, "
"note down"
No space before comma.
"this is a lot of work"